hw/ip/adc_ctrl/doc/_index.md

```diff
@@ -81,22 +81,23 @@ Note that the time taken in this step depends on the properties of the ADC.
 
 In active operation the controller is in continuous scanning mode:
 * The ADC is continually powered on.
-* The sampling cycle time is the time taken for the ADC to take two samples (450us) plus NNN (fill in based on implementation but probably 4-10)) number of slow clock cycles x (5us; always-on clock frequency is 200KHz).
-* The debounce timer will trigger the filter status (and interrupt) after {{< regref "adc_sample_ctl" >}} multiplied by the continuous sampling cycle time.
+* The sampling cycle time is the time taken for the ADC to take two samples (450us) plus processing time from the adc controller..
```

Comment:
Line 72 above is missing a closing parenthesis.

Reply:
wow how did you see that?

Reply:
I think from looking at the rendered document...